config.js

```diff
@@ -73,6 +73,8 @@ const DEFAULT_PERSIST_REDIS_SAMPLE_STORE_MILLISECONDS = 120000; // 2min
 
 const botEventLimit = pe.BOT_EVENT_LIMIT || 100;
 
+const deactivateRoomsInterval = pe.DEACTIVATE_ROOMS_INTERVAL || 300000; // 5min
```

Comment:
should be `+pe.DEACTIVATE_ROOMS_INTERVAL` which coerces that env var string to number